gems/aws-sdk-core/lib/aws-sdk-core/shared_config.rb

```diff
@@ -283,7 +285,7 @@ def validate_profile_exists(profile)
       unless (@parsed_credentials && @parsed_credentials[profile]) ||
           (@parsed_config && @parsed_config[profile])
         msg = "Profile `#{profile}' not found in #{@credentials_path}"
-        msg << " or #{@config_path}" if @config_path
+        msg += "or #{@config_path}" if @config_path
```

Comment:
Same comment on relation to the PR intention.

Reply:
yes, this one is directly related to the change we're doing to the string in the line above